### PR TITLE
Fix es

### PR DESCRIPTION
**Pull request by sabban**

### CODE REVIEW

File: nxtool/log_providers/elastic.py [hunk 1]
```diff
@@ -191,9 +191,14 @@ def commit(self):
             for x in entry.keys():
                 if isinstance(entry[x], basestring):
                     entry[x] = unicode(entry[x], errors='replace')
-                items.append(entry)
+            items.append(entry)
             count += 1
-        self.client.bulk(body=items)
+        try:
+            self.client.bulk(body=items)
+        except:
```
Comment (jvoisin) [inline nxtool/log_providers/elastic.py]: Please specify the exception :3

File: nxtool/log_providers/elastic.py [hunk 2]
```diff
@@ -191,9 +191,14 @@ def commit(self):
             for x in entry.keys():
                 if isinstance(entry[x], basestring):
                     entry[x] = unicode(entry[x], errors='replace')
-                items.append(entry)
```
Comment (jvoisin) [inline nxtool/log_providers/elastic.py]: Are you sure about this change?

Comment (sabban) [inline nxtool/log_providers/elastic.py]: Yes, elasticsearch api corrected this mistake by deleting duplicate events...

File: nxtool/log_providers/elastic.py (hunk 1 above)
Comment (sabban) [inline nxtool/log_providers/elastic.py]: You are right. Eventually I will get used good practices.

File: nxtool/log_providers/elastic.py (hunk 2 above)
Comment (jvoisin) [inline nxtool/log_providers/elastic.py]: Ok :)

**Pull request merged**